integration-tests/test/basic-l1-l2-communication.spec.ts

```diff
@@ -129,78 +156,41 @@ describe('Basic L1<>L2 Communication', async () => {
             gasLimit: DEFAULT_TEST_GAS_L1,
           }
         )
+
       const receipt = await tx.wait()
 
       const waitUntilBlock =
         receipt.blockNumber + envConfig.DTL_ENQUEUE_CONFIRMATIONS
-      let currBlock = await env.l1Provider.getBlockNumber()
+      let currBlock = await env.messenger.l1Provider.getBlockNumber()
       while (currBlock <= waitUntilBlock) {
         const progress =
           envConfig.DTL_ENQUEUE_CONFIRMATIONS - (waitUntilBlock - currBlock)
         console.log(
           `Waiting for ${progress}/${envConfig.DTL_ENQUEUE_CONFIRMATIONS} confirmations.`
         )
         await sleep(5000)
-        currBlock = await env.l1Provider.getBlockNumber()
+        currBlock = await env.messenger.l1Provider.getBlockNumber()
       }
       console.log('Enqueue should be confirmed.')
 
       await awaitCondition(
         async () => {
           const sender = await L2SimpleStorage.msgSender()
-          return sender === env.l1Wallet.address
+          return sender === (await env.messenger.l1Signer.getAddress())
         },
         2000,
         60
       )
 
       // No aliasing when an EOA goes directly to L2.
-      expect(await L2SimpleStorage.msgSender()).to.equal(env.l1Wallet.address)
-      expect(await L2SimpleStorage.txOrigin()).to.equal(env.l1Wallet.address)
-      expect(await L2SimpleStorage.value()).to.equal(value)
-      expect((await L2SimpleStorage.totalCount()).toNumber()).to.equal(1)
-    })
-
-    it('should have a receipt with a status of 1 for a successful message', async () => {
-      const value = `0x${'42'.repeat(32)}`
-
-      // Send L1 -> L2 message.
-      const transaction = await env.l1Messenger.sendMessage(
-        L2SimpleStorage.address,
-        L2SimpleStorage.interface.encodeFunctionData('setValue', [value]),
-        5000000,
-        {
-          gasLimit: DEFAULT_TEST_GAS_L1,
-        }
-      )
-      await transaction.wait()
-
-      const { remoteReceipt } = await env.waitForXDomainTransaction(
-        transaction,
-        Direction.L1ToL2
-      )
-
-      expect(remoteReceipt.status).to.equal(1)
-    })
-
-    // SKIP: until we decide what should be done in this case
-    it.skip('should have a receipt with a status of 0 for a failed message', async () => {
```

Comment:
I removed this test because we should just handle this as part of 1.0. No need to have a dead test here for behavior we haven't specified.